public/js/duplicateCheck.js

```diff
@@ -0,0 +1,221 @@
+/**
+* PEP Capping 2017 Algozzine's Class
+*
+* Duplicate match search that looks for possible duplicates in the db based on user input into 
+* intake, self-referral, or agency-referral packets.
+*
+* Ussage: When an employee fills out an intake/self-referral/agency-referral form they should not 
+* be creating a new participant with each form. This function grabs the user supplied input from whichever
+* form they are filling out, and populates it into a hidden form. That hidden form is then submitted through an
+* ajax request which routes to a page with a sql query that looks for any possible matches in the db.
+* If matches are found, the result is displayed to the user via a bootstrap modal. From there, the user
+* Will be able to scroll through the possible matches and select the match they believe to be the participant 
+* they are filling out information for. If there are no matches or if the user determines they want to create
+* a new participant, the modal will close/not appear and submit the form as a new participant.
+*
+* @author Vallie Joseph
+* @copyright 2017 Marist College
+* @version 1.1
+* @since 1.1
+*
+*
+*/
+
+$(document).ready(function(){
+
+var pathname = window.location.pathname; // Returns path only
+var firstNameValue;
+var middleinit ;
+var lastname;
+var pers_primphone ;
+var address ;
+var apt;
+var zip;
+var city ;
+var state ;
+var dob ;
+var race;
+var sex;
+
+/**
+* Checking the page route to see what page the
+* user is on; this will tell the modal what js function
+* to run depending on the form that is being completed
+* @params pathN string that contains the path route
+*/  
+function routePage(pathN){
+    if(pathN == "/referral-form"){
+        submitAll();
+    }else if ( pathN == "/self-referral-form"){
+        submitAllSelf()
+    }else if( pathN == "/intake-packet"){
```

Comment:
I would suggest sticking to one style of spacing.

Either (line 49):
```php
}else if ( pathN == "/self-referral-form"){
```
or (line 51):
```php
}else if( pathN == "/intake-packet"){
```

Personally I'd suggest to at least put a space before the `else` keyword.